src/scss/_display-controls.scss

```diff
@@ -12,9 +12,9 @@
   transition: transform 0.2s;
   z-index: 1;
 
-  .dark-mode & {
-    background-color: $sttm-light-black;
-  }
+  // .dark-mode & {
```

Comment:
We need this here, currently, dark mode is like this

![image](https://user-images.githubusercontent.com/1990932/84476359-a1134080-acab-11ea-88ee-68c28c50f9d0.png)

Reply:
ahh good catch thanks !